app-error-handlers.js

```diff
@@ -0,0 +1,21 @@
+exports.customErrorHandler = ((err, req, res, next) => {
+    if(err.status && err.message) {
+        res.status(err.status).send({msg: err.message})
+    } else{
+        next(err)
+    }
+})
+
+
+
+exports.psqlErrorHandler = (err, req, res, next) => {
+    if (err.code === '22P02') {
+        res.status(400).send({message: 'invalid id type'})
+    }else{
+        next(err)
+    }
+}
+
+exports.serverErrorHandler = (err, req, res, next) => {
+    res.status(500).send({message: 'internal server error'})
+}
```

Comment:
A lot of error handlers have been set up here. These will cause your future tests to pass straight away which ideally we don't want! Please do delete these for now. Feel free to keep a note of them somewhere though 😄